### PR TITLE
HOL1-5 WMS ID : 11147, 11150, 11151, 11152, 11153

### PR DESCRIPTION
**Pull request by tomohamm**
# 11147, 11150, 11151, 11152, 11153

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: apex-professional-hol2/4-creating-db-objects-quicksql/4-creating-db-objects-quickSQL.md [hunk 1]
```diff
@@ -37,95 +38,123 @@ Estimated Time: 15 minutes
...
+
+    ![Check Table Employee](images/check-table-employee.png " ")
 
-12. Click the **Data** tab to see the data that was loaded.
+5. Click **Create Blueprint**.
 
-    ![View department table](images/view-departments-table-data1.png " ")
+6. Click **Preview Data** to preview data for the Employee table and columns.
 
-## Task 2: Load data using the Data Workshop Utility
```
Comment (hope-fisher) [inline apex-professional-hol2/4-creating-db-objects-quicksql/4-creating-db-objects-quickSQL.md:87]: renumber as Task 3

File: apex-professional-hol4/creating-order-page.md [hunk 2]
```diff
@@ -12,127 +12,140 @@ Customers will find the following details of the order:
...
-2. Right click the region and click **Create Sub Region**.
+1. In the Rendering tree (left pane), right-click on the **Thank you for your Order!** region and select **Create Sub Region**.
 
-    ![Create Sub Region](./images/create-sub-region11.png " ")
+    ![Create Sub Region](./images/create-sub-region1.png " ")
 
-3. In the Property Editor, enter the following:
-    - For Title - enter the expression (including the period) **Order: &P15_ORDER.**
-    - For Type - select **Static Content**  
+3. In the Property Editor, enter/select the following:
```
Comment (hope-fisher) [inline apex-professional-hol4/creating-order-page.md:130]: numbering. This step in task 6 should be step 2.

**Review by hope-fisher — COMMENTED** [review]
Please see individual comments for small changes needed. In the future, please submit only one workshop per PR to facilitate quicker review.

**Comment by tomohamm** [comment]
Thanks for your comments. I have resolved them.

**Review by hope-fisher — APPROVED** [review]

**Pull request merged**